templates/base/head.tmpl

```diff
@@ -49,6 +49,8 @@
 			i18n: {
 				copy_success: '{{.i18n.Tr "copy_success"}}',
 				copy_error: '{{.i18n.Tr "copy_error"}}',
+				error_occurred: '{{.i18n.Tr "error.occurred"}}',
+				issue_not_found: '{{.i18n.Tr "error.issue_not_found"}}',
```

Comment:
I hope we won't fill the `i18n` to a fat map. `issue_not_found` is used only once, it can be put in to AJAX response.

Reply:
Is their a current API that gives you the translation of specified strings? Otherwise IMO to add a new single endpoint for that, it will just make it more bloated.

Also, how to handle when the network request fails(e.g. temporary wifi loss)?

Reply:
I get some ideas and will try to propose a change on this PR. Give me some time.  :D